### ISSUE TITLE
POST to read-only portfolios should return an error

### ISSUE THREAD

**Issue by spigo900** [spigo900/abet-submission-system#20]
**[FR8]**
**Description:**
As a DUS, I want course portfolios' read-only status to be updated and checked before accepting a portfolio update request via POST so that instructors cannot edit course portfolios whose expiration date has passed.

Part of the epic in issue #17.

**Priority:** Medium
**Effort:** 1
**Acceptance Criteria:**
1. [x] When a POST request is made to change a course portfolio (on `/course/:id` with `id='edit_course_form'`), the portfolio's read-only status is updated.
2. [x] The portfolio's read-only status is checked only after it is updated.
3. [x] If the portfolio is marked read-only, an error message should be returned.

**Comment by spigo900** [comment]
Note to self: It is definitely `POST` that needs to respond this way and not, say, `PATCH` because `POST` is the method that the form uses. Our form might be using the wrong HTTP method but that's a separate issue.

**Comment by spigo900** [comment]
I tested this in Firefox 70.0.1 (64-bit) on Windows 10.
1. I opened the website's log-in page, logged in, and clicked the "view" link for a read-only portfolio.
2. I opened the inspector (F12) and removed the "disabled" attribute on the submit button.
3. I clicked the "Submit" button at the bottom of the form.
    a. I expected to get an error page telling me that the portfolio was marked read-only.
    b. I got the expected error page.
